src/Calculator.java

```diff
@@ -1,36 +1,38 @@
 
 public class Calculator {
 	public static void main(String[] args) {
-		int ra = Add(4, 2);
-		int rs = Sub(4, 2);
-		int rm = Multiply(4, 2);
-		int rd = Div(4, 2);
+		Calculator calculator = new Calculator();
+		
+		int ra = calculator.Add(4, 2);
+		int rs = calculator.Sub(4, 2);
+		int rm = calculator.Multiply(4, 2);
+		int rd = calculator.Div(4, 2);

```

Comment:
replace ra, rs, rm and rd with proper variable names like additionResult, substractionResult, etc